### ISSUE TITLE
Executive Summary in NetApp OpenStack Docs

### ISSUE THREAD

**Issue by forgosh** [NetApp-openstack-dev/openstack-docs#2]
While I applaud the direction of making the DOG a living document which is out of date as soon as printed, some USPS customers do not have internet access much of the time due to security restrictions. These customers will need a way to have a hard copy.

This bug tracker is for errors with the documentation, use the following as a template and remove or add fields as you see fit. Convert [ ] into [x] to check boxes:

- [ ] This doc is inaccurate in this way: ______
- [ ] This is a doc enhancement request.
- [ ] I have a fix to the document that I can paste below including example: input and output. 

If you have a troubleshooting or support issue, use the following  resources:

 - thePub NetApp Developer Forum: http://netapp.io/
 - Slack: http://netapp.io/slack/
 - IRC: 'openstack-netapp' channel on Freenode

-----------------------------------
Release: 2017.1 on 2017-09-08 19:07
SHA: 2b1394dee807ed4dbd4c684e39eda7c5efdbc00f
Source: https://github.com/NetApp-openstack-dev/openstack-docs/ch_executive-summary.rst
Current URL: https://netapp-openstack-dev.github.io/openstack-docs/ocata/ch_executive-summary.html
Filed against: OCATA

**Comment by korenaren** [comment]
Understood Seth. Will try to find a way for users to print the rst, or a way for us to include a pdf.

**Comment by adkerr** [comment]
http://netapp.github.io/openstack-deploy-ops-guide/ocata/openstack-deployment-ops-guide.pdf

Andrew Kerr
Test Automation Engineer
Open EcoSystems

NetApp
919.476.4899 Direct Phone
Andrew.Kerr@netapp.com
________________________________

**Comment by adkerr** [comment]
To elaborate on my previous post, pdfs are already auto generated by the doc build system. Just replace `ocata` with the release version you want in the above URL.

**Comment by forgosh** [comment]
Ok. So we just need a link to it for those users

Sent from my iPhone

On Sep 11, 2017, at 4:54 PM, Andrew Kerr <notifications@github.com<mailto:notifications@github.com>> wrote:


To elaborate on my previous post, pdfs are already auto generated by the doc build system. Just replace ocata with the release version you want in the above URL.

**Comment by korenaren** [comment]
Need @gouthampacha to verify. Not sure those are built from the new RST format.

**Comment by gouthampacha** [comment]
@korenaren @forgosh: Yes, the new documentation build does not support pdf builds yet. 

We had discussed in the past to disallow downloading these docs, because these are living documents. They do get updated if we find any issues, or decide to add any more content. However, you make a good case of customers who may not have internet access. We'll get the PDF build working. We're bikeshedding on one right now in our internal code review system, but we'll probably post the link right out of the document: https://netapp-openstack-dev.github.io/openstack-docs/ocata soon!
